packages/components/tests/dummy/app/styles/app.scss

```diff
@@ -26,14 +26,15 @@
 
 // Notice: this list can be automatically edited by the Ember blueprint, please don't remove the start/end comments
 // START COMPONENT PAGES IMPORTS
+@import "./showcase-pages/tooltip";
```

Comment:
I thought the documentation page should stay as "tooltip" instead of referencing the updated component name (TooltipButton).